client/src/modules/list/components/ItemsList/ItemsList.jsx

```diff
@@ -41,24 +41,20 @@ class ItemsList extends Component {
   };
 
   voteForItem = item => () => {
-    const { _id, voterIds } = item;
+    const { _id, isVoted } = item;
     const {
-      vote,
-      currentUser: { id },
+      clearVote,
+      setVote,
       match: {
         params: { id: listId }
       }
     } = this.props;
-    voterIds.includes(id)
-      ? vote(_id, listId, voterIds.filter(voterId => voterId !== id))
-      : vote(_id, listId, voterIds.concat(id));
+
+    isVoted ? clearVote(_id, listId, isVoted) : setVote(_id, listId, isVoted);
```

Comment:
- remove `isVoted` from function calls - this is not needed anymore
- please do not use ternary operator if return value is not needed
- or:
```
const action = isVoted ? clearVote : setVote;
action(_id, listId);
```